recipes/laslib/all/patches/0001-build-only-laslib.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/LASlib/CMakeLists.txt b/LASlib/CMakeLists.txt
+index 5efb193..6037262 100644
+--- a/LASlib/CMakeLists.txt
++++ b/LASlib/CMakeLists.txt
+@@ -9,4 +9,4 @@ else()
+ endif()
+ 
+ add_subdirectory(LASlib/src)
+-add_subdirectory(LASlib/example)
++
```

Comment:
```suggestion
```
This part of the patch should no longer be necessary since the `LASlib/example/CMakeLists.txt` is now truncated in the build step.